### DISCUSSION TITLE
7zip installer cleanup

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [microsoft/winget-pkgs#34046] [Ideas]
7zip package is a mess. it should be cleaned up.

**Comment by vedantmgoyal9** [comment]
I think thats because exe and msi version have different version in ARP, exe has ad 6.3.3 and msi had 6.3.3, winget upgrade wont work properly if both of them are in same file, same issue with 7zip. I think manifest v1.1.0 might solve this issue.

From: https://github.com/microsoft/winget-pkgs/discussions/33494#discussioncomment-1568391

Manifest schema v1.1 would fix it.
